Cargo.lock

```diff
@@ -677,9 +677,9 @@ dependencies = [
 
 [[package]]
 name = "napi"
-version = "2.13.1"
+version = "2.13.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "d7f0a2e93526dd9c8c522d72a4d0c88678be8966fabe9fb8f2947fde6339b682"
+checksum = "0ede2d12cd6fce44da537a4be1f5510c73be2506c2e32dfaaafd1f36968f3a0e"
```

Comment:
Not sure why the version was changed here, while we are still using the old version in `Cargo.toml`. Is this intentional?

Cargo should have errored out here. I think I'm missing something.

Reply:
no idea, not my doing.

Reply:
I see it was removed from this PR. Should be fine now.